components/app/PlaceItem.jsx

```diff
@@ -85,11 +88,13 @@ const PlaceItemContainer = ({ children, ...props }) => (
     {children}
   </Box>
 );
-const PlaceItem = ({ children, place: { name, category, source, tags }, ...props }) => (
+const PlaceItem = ({ children, place: { name, category, source, tags, collection, delivery }, ...props }) => (
   <PlaceItemContainer {...props}>
+  <Link to={`/${_.kebabCase(name)}`}>
```

Comment:
`components` should for universal React code, so platform/framework specific imports like `<Link>` will break the `Figma` export, React Native, etc. I would create a new `Link` component with something like this:

```js
const Link = Platform.select({
  web: require('gatsby').Link,
  figma: React.Fragment
})
```

I understand that this adds extra friction/confusion :/ , but by doing this, this adds future proofing for having a React Native app in the future, or using the Figma component exports.

Reply:
ah ok, so we shouldn't do Link because it is Gatsby specific?

Reply:
I'll commit an example. The idea is to use components that work cross-platform, or to create a new cross-platform component, to support `react-native-web`, `react-figma`, `react-sketchapp`, etc.

example using platform filename resolving (when you import, the relevant platform file is imported):

`components/Link/index.web.js`

```js
export { Link as default } from 'gatsby';
```

`components/Link/index.figma.js`

```js
export { Fragment as default } from 'react';
```

Then you can do `import Link from '../Link';`.

I hope this makes sense. This isn't a common practice, but is necessary for the `react-figma` app to work, but the idea is that most of this work should be abstracted away, say by releasing a `react-link-primitives` package to `npm`, resulting in a cross-platform component library that magically works on all platforms 🙂.

Reply:
Implemented/documented in b1c1956